### PR TITLE
Modifed the design of comment like API and added the likedCommentsByuserAndCourseId API

### PR DESCRIPTION
**Pull request by ruksana2808**
…serAndCourseId API

### CODE REVIEW

File: src/main/java/com/tarento/commenthub/service/impl/CommentServiceImpl.java [hunk 1]
```diff
@@ -659,6 +644,34 @@ public ApiResponse deleteReportedComments(Map<String, Object> request, String to
...
+  @Override
+  public ApiResponse getCommentsLikedByUser(String courseId, String token) {
+    log.info("CommentServiceImpl:getCommentsLikedByUser::inside the method");
+    ApiResponse response = new ApiResponse();
+    response.setResponseCode(HttpStatus.OK);
+    String userId = accessTokenValidator.verifyUserToken(token);
+    if (StringUtils.isBlank(userId) || userId.equalsIgnoreCase(Constants.UNAUTHORIZED_USER)) {
+      return returnErrorMsg(Constants.INVALID_USER, HttpStatus.BAD_REQUEST, response);
+    }
+    if(StringUtils.isBlank(courseId)){
```
Comment (Sahil-tarento) [inline src/main/java/com/tarento/commenthub/service/impl/CommentServiceImpl.java]: Please do formatting

File: src/main/java/com/tarento/commenthub/service/impl/CommentServiceImpl.java [hunk 2]
```diff
@@ -330,90 +337,68 @@ public ApiResponse likeComment(Map<String, Object> likePayload) {
...
-          commentToBeUpdated.setCommentData(commentData);
-          Comment updatedComment = commentRepository.save(commentToBeUpdated);
-          if (likePayload.containsKey(likePayload.get(Constants.COMMENT_TREE_ID))
-              && StringUtils.isBlank((String) likePayload.get(Constants.COMMENT_TREE_ID))
-              && likePayload.get(Constants.COMMENT_TREE_ID) != null) {
-            deleteRedisKey(
-                generateRedisJwtTokenKey((String) likePayload.get(Constants.COMMENT_TREE_ID),
-                    defaultOffset, defaultLimit));
+            Long incrementCount = commentData.get((String) likePayload.get(Constants.FLAG))
+                .asLong();
```
Comment (Sahil-tarento) [inline src/main/java/com/tarento/commenthub/service/impl/CommentServiceImpl.java:372]: Please do a proper check

**Review by Sahil-tarento — CHANGES_REQUESTED** [review]

File: src/main/java/com/tarento/commenthub/service/impl/CommentServiceImpl.java (hunk 1 above)
Comment (ruksana2808) [reply src/main/java/com/tarento/commenthub/service/impl/CommentServiceImpl.java]: done

File: src/main/java/com/tarento/commenthub/service/impl/CommentServiceImpl.java (hunk 2 above)
Comment (ruksana2808) [reply src/main/java/com/tarento/commenthub/service/impl/CommentServiceImpl.java:372]: done

**Pull request merged**